qiita_db/sql_connection.py

```diff
@@ -150,16 +150,32 @@ def _open_connection(self):
 
         try:
             self._connection = connect(**args)
-        except Exception:
-            # catch any exception and raise as runtime error
-            # attempt to point the user to their config file
-            error_text = ("Can't connect to database; user `%s` is not known. "
-                          "Your QIITA configuration file, located at `%s` must"
-                          " specify the correct user, via the `USER` parameter"
-                          " under the `[postgres]` header. Please refer to "
-                          "`INSTALL.md` in the QIITA base directory.")
-            error_text = error_text % (qiita_config.user, qiita_config.conf_fp)
-            raise RuntimeError(error_text)
+        except Exception as e:
```

Comment:
If we are able to find the errors by the exceptions then we should totally use the messages you have below!